src/components/GetInvolvedCard.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import { Box, Text, Flex } from '@chakra-ui/react';
+import { graphql, Link } from 'gatsby';
+import Img from 'gatsby-image';
+
+interface Props {
+  data: GatsbyTypes.GetInvolvedCardFragment;
+}
+
+const GetInvolvedCard = ({ data }: Props) => {
+  const { title, icon, link } = data;
+
+  const imageEl =
+    icon?.fluid != null ? <Img fluid={icon.fluid} imgStyle={{ objectFit: 'contain' }} /> : null;
+
+  return (
+    <Box w={500} my={10}>
```

Comment:
Can we make the entire box the `Link` so it's all clickable? I might also talk to design about a potential hover state